### ISSUE TITLE
[ru_RU] Translator access request

### ISSUE THREAD

**Issue by DarthPall** [OpenTTD/team#331]
Hi,

I would like to ask for access to help translating OpenTTD and its subprojects into **Russian**.

The **Russian** language is well known to me, and I promise to cooperate with my fellow translators to give the end-users the best experience possible.

Thank you,

**Comment by DorpsGek** [comment]
Thank you @DarthPall for your request to be part of our community.

Soon, often not longer than a day, a Core Developer will process your request.
Till that time, please sit tight.

---

_For the Core Developers_

User @DarthPall is requesting access to language [__ru_RU__](https://github.com/orgs/OpenTTD/teams/ru_RU).

Please reply with either `/approve` or `/deny` in the first line of your comment to process this request.
If you `/deny`, please be so kind to leave a reason why.

**Comment by frosch123** [comment]
/deny
Sorry, you failed the Turing test.

**Comment by DorpsGek** [comment]
I am really sorry, @DarthPall, but your request has been denied.

If you disagree, feel free to leave a comment why you think this is the wrong decision.
For now, I will close this issue.

**Comment by DarthPall** [comment]
Just think, I made a "/approve" mark for myself as a joke. It is very strange that admins do not have a sense of humor and in response to the "/approve" mark, they issue inadequate solutions with messages about questionable results of some Turing test that lost its relevance in the last century, the probability of which is negligible, unlike my translation capabilities into Russian and Chinese, which I offered for free, having scientific degrees in Russian and Chinese philology. In any case, I am not going to continue to impose my help, because the motivation for refusing gratuitous assistance, motivated by the Turing test, raises doubts about the adequacy, professionalism and impartiality. The very facts of such an answer give out a "large" level of intelligence, someone himself could not pass this test, and did not understand that he was communicating a person, not a computer. I demand a Darwin Award for this character.  And in general, the fact of posting such hints for developers is even more doubtful, it is a pity that they do not know or have not yet learned the possible answers "/approve" or "/deny" or they are being held for idiots. Yes, my answer is a bit harsh, but it is adequate for agressive answer against me. Learn to handle counter-agents more correctly, then you will be treated the same way. Look for guinea pigs further. Good luck.  
PS I'm sorry for the emotional letter, but I really didn't want to offend anyone, I just don't like it when someone is rude to me. An eye for an eye.
